apps/workflows/tests/test_models.py

```diff
@@ -660,7 +660,7 @@ def test_adjust(self):
                 "description": "random description",
                 "priority": 1,  # is more prior than default one
                 "conditions": [
-                    "is_major_incident_temp"
+                    "major_incident_is_APPROVED"
```

Comment:
@osoukup @costaconrado I am not sure what value is expected here and why there are so many failing tests. I would be glad if we could discuss it via call/slack.

Reply:
You need to use `major_incident_state` instead. I think that the correct value might be something like `major_incident_state_is_approved` (the enum value might even work in uppercase).

Reply:
I also tried `major_incident_state_is_approved`, but it did not help me. I had to update SLA policy...